downstream/modules/platform/ref-database-inventory-variables.adoc

```diff
@@ -5,11 +5,122 @@
 [cols="50%,50%,50%",options="header"]
 |====
 | *RPM variable name* | *Container variable name* | *Description*
+//EDA database variables
+| `automationedacontroller_install_pg_host` | `eda_` | Configure the PostgreSQL host for {EDAName}, which can be an externally managed database. 
```

Comment:
Just a suggestion but for the sake of clarity I'd propose listing out the full variable name as opposed to just `eda_` for example as this may be confusing for the user